x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_notify_when.tsx

```diff
@@ -328,9 +328,9 @@ export const ActionNotifyWhen = ({
                       name="throttle"
                       data-test-subj="throttleInput"
                       prepend={i18n.translate(
-                        'xpack.triggersActionsUI.sections.ruleForm.ruleNotifyWhen.label',
+                        'xpack.triggersActionsUI.sections.ruleForm.frequencyNotifyWhen.label',
                         {
-                          defaultMessage: 'Every',
+                          defaultMessage: 'Run action every',
```

Comment:
<img width="568" alt="Screenshot 2023-05-04 at 3 43 05 PM" src="https://user-images.githubusercontent.com/1445834/236324840-67b65893-ad73-441d-ab34-30ab5c66c453.png">

We need to revert this until we get rid of the flyout